### PR TITLE
Bug bitbucket

### PR DESCRIPTION
**Pull request by bush4u**

### CODE REVIEW

File: components/bitbucket/sources/common/common.mjs [hunk 1]
```diff
@@ -72,6 +72,7 @@ export default {
     async deploy() {
       // Retrieve historical events
       const events = await this.loadHistoricalData();
+      console.log("events", events);
```
Comment (lcaresia) [inline components/bitbucket/sources/common/common.mjs:75]:
```suggestion
```

**Review by lcaresia — CHANGES_REQUESTED** [review]
Just a minor fix!

**Comment by bush4u** [comment]
> 

Done

**Review by lcaresia — APPROVED** [review]
LGTM!

**Pull request merged**